### PR TITLE
Fixing issue when splitting trie nodes

### PR DESCRIPTION
**Pull request by jcurbelo**
Fixing bug having only a file while splitting a trie node

### CODE REVIEW

**Comment by pbu88** [comment]
Looks great! Good catch, Thank you!

File: treeFunctions.js [hunk 1]
```diff
@@ -72,8 +70,24 @@ function _insert(tree, dirs, file) {
...
+
+        // If the new subtree is a file
+        if (dirsSuffix == ''){
+            // Appending the new leaf
+            leaf = createLeaf(tree, file);
+            tree.files.push(leaf);
+            return;            
+        }
+
+
```
Comment (pbu88) [inline treeFunctions.js]: No need for two empty lines but is alright :)

Comment (pbu88) [inline treeFunctions.js]: Thanks btw!

Comment (jcurbelo) [inline treeFunctions.js]: You're welcome!

**Pull request merged**